extensions/skins/conversation_v1/Conversation.js

```diff
@@ -18,41 +18,184 @@
  * @author sll@google.com (Sean Lip)
  */
 
+var TIME_FADEOUT_MSEC = 100;
+var TIME_HEIGHT_CHANGE_MSEC = 500;
+var TIME_FADEIN_MSEC = 100;
+var TIME_NUM_CARDS_CHANGE_MSEC = 500;
+
+oppia.animation('.conversation-skin-animate-cards', function() {
+  // This removes the newly-added class once the animation is finished.
+  var animateCards = function(element, className, done) {
+    var tutorCardElt = jQuery(element).find('.conversation-skin-tutor-card');
+    var supplementalCardElt = jQuery(element).find(
+      '.conversation-skin-supplemental-card');
+
+    if (className === 'animate-to-two-cards') {
+      supplementalCardElt.css('opacity', '0');
+      tutorCardElt.animate({
+        'margin-left': '0'
+      }, TIME_NUM_CARDS_CHANGE_MSEC, function() {
+        tutorCardElt.css('margin-left', '0');
+        tutorCardElt.css('margin-right', '');
+        tutorCardElt.css('float', 'left');
+
+        supplementalCardElt.animate({
+          'opacity': '1'
+        }, TIME_FADEIN_MSEC, function() {
+          supplementalCardElt.css('opacity', '1');
+          jQuery(element).removeClass('animate-to-two-cards');
+
+          tutorCardElt.css('margin-left', '');
+          tutorCardElt.css('margin-right', '');
```

Comment:
The same instruction is given on line 39. I just want to make sure this line is also necessary. Is margin-right being changed by anything else here?